.github/workflows/ci.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches: [main]
     tags: ["releases/**"]
   pull_request:
-    branches: ["*"]
+    branches: ["**"]
```

Comment:
I added this so that CI runs on my branches with a slash in the branch name. I'm happy to remove this if you're not interested in this change.

Reply:
🤔 I've been using slashes in my branch names and didn't have an issue with CI running: https://github.com/foxglove/schemas/pull/158

Reply:
This PR is targeting a branch other than `main` (my other branch, `fgwt202412/ubuntu-22_04-python-3_7), which does have a slash in it.

[Docs](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore)

Reply:
ah, makes sense, this is for the target branch rather than the head branch. In some other repos we use `pull_request: {}` which may do the same thing by default? Anyhow, this seems fine